lms/envs/aws.py

```diff
@@ -917,6 +917,11 @@
     ENTERPRISE_ENROLLMENT_API_URL
 )
 
+# Enterprise logo image size limit in KB's
+ENTERPRISE_CUSTOMER_LOGO_IMAGE_SIZE = ENV_TOKENS.get(
+    'ENTERPRISE_CUSTOMER_LOGO_IMAGE_SIZE'
+)
```

Comment:
should we be falling back on whatever is set if there is no override? it seems we should add another argument here with the original variable just as other settings are.